src/Service/Notifier/SlackNotifier.php

```diff
@@ -17,15 +17,17 @@
 
 readonly class SlackNotifier implements ChannelNotifierInterface
 {
+    private const CHANNELS = ['bugs' => 'C02UDN5PZFZ', 'features' => 'C05TWL4N3R9'];
+
     public function __construct(private ChatterInterface $chatter, private RouterInterface $router, private LoggerInterface $logger)
     {
     }
 
-    private function createSlackOptions(Bug $bug, string $text): SlackOptions
+    private function createSlackOptions(Bug $bug): SlackOptions
     {
         return (new SlackOptions())
             ->iconEmoji($bug->isDone() ? 'white_check_mark' : 'Bug')
-            ->username(sprintf('[MPB] %s', $text))
+            ->username(sprintf('[MPB] %s', $bug->isDone() ? 'Bug résolu' : 'Nouveau bug'))
             ->block((new SlackSectionBlock())->text($bug->getTitle() ?? ''))
```

Comment:
There are 3 occurences of `$bug->isDone() ? 'Bug résolu' : 'Nouveau bug'`, it may needs a little refacto